.vib/redmine/cypress/cypress/fixtures/issues.json

```diff
@@ -0,0 +1,7 @@
+{
+  "newIssue": {
+    "tracker": "Bug",
```

Comment:
It seems we are not using `newIssue.tracker`.

Reply:
True. I added the test data thinking I'll need it. Then I forgot to remove it. Now it's fixed.